### DISCUSSION TITLE
Keyboard shortcut for saving all changes

### DISCUSSION THREAD

**Discussion by mongedelonge** [cables-gl/cables_docs#286] [Feature Requests] [answered]
It would be helpful to have a keyboard shortcut for saving every unsaved change, like simultaneously saving text editor tabs and the patch. It takes extra work to save the edits to text/shaders/etc and then saving the patch, and if you forget to save the patch some work can be lost.

**Comment by steam0r** [comment, marked as answer]
we actually decided against doing this for a few reasons, one being its not that easy to do since we have various things happening when op-code is saved (like executing the op, and checking for errors and the likes) and another one being that we would then have to be really careful to not save your patch in a broken state somehow. 

we think its more clear to keep code editing and patching "seperate", even if visually happening in the same window, for now.

**Reply by mongedelonge** [reply]
fair enough. thanks
